nix/modules/kld/default.nix

```diff
@@ -209,7 +218,7 @@ in
         KLD_DATABASE_CLIENT_CERT_PATH = lib.mkDefault "/var/lib/kld/certs/client.kld.crt";
         KLD_DATABASE_CLIENT_KEY_PATH = lib.mkDefault "/var/lib/kld/certs/client.kld.key";
         KLD_EXPORTER_ADDRESS = lib.mkDefault cfg.exporterAddress;
-        KLD_REST_API_ADDRESS = lib.mkDefault cfg.restApiAddress;
+        KLD_REST_API_ADDRESS = if lib.lists.any (_: true) cfg.apiIpAccessList then "0.0.0.0:${toString cfg.restApiPort}" else "127.0.0.1:${toString cfg.restApiPort}";
```

Comment:
```suggestion
        KLD_REST_API_ADDRESS = if cfg.apiIpAccessList != [] then "0.0.0.0:${toString cfg.restApiPort}" else "127.0.0.1:${toString cfg.restApiPort}";
```

Reply:
You can even see if `"[::]:${toString cfg.restApiPort}"` works instead of 0.0.0.0 (this listens to dual stack addresses)

Reply:
I am not sure the exactly reason `[::]:${toString cfg.restApiPort}` is not work here, so I keep using `0.0.0.0:${toString cfg.restApiPort}`